index-scheduler/src/utils.rs

```diff
@@ -572,3 +574,30 @@ pub fn dichotomic_search(start_point: usize, mut is_good: impl FnMut(usize) -> b
         }
     }
 }
+
+pub struct Bincode<T>(std::marker::PhantomData<T>);
```

Comment:
Can you add more documentation, please?

Reply:
Same, we won’t need this if we wait until bincodev2 is officially released